Experimental/Asteroids-CMB-NGO-Sample/Assets/Scenes/SmallDemos/Scripts/ObjectSpawner.cs

```diff
@@ -158,6 +158,10 @@ private void Update()
 
     private void UpdateInput()
```

Comment:
I see that there was a previous version of this method below. We could remove that version?